src/researcher.js

```diff
@@ -77,6 +78,7 @@ var Researcher = function( paper ) {
 		getSentenceBeginnings: getSentenceBeginnings,
 		relevantWords: relevantWords,
 		sentences,
+		h1s: h1s,
```

Comment:
You don't have to provide a key if it is the same as a variable. Under the hood javascript is doing this.